runtime/oti/j9nonbuilder.h

```diff
@@ -5548,6 +5548,13 @@ typedef struct J9JavaVM {
 	jfieldID criuSupportJVMRestoreFailure;
 	jclass criuResultClass;
 	jmethodID criuResultInit;
+	jclass criuSupportClass;
+	jfieldID criuSupportCheckPointDir;
+	jfieldID criuSupportKeepRunning;
+	jfieldID criuSupportShellJob;
+	jfieldID criuSupportExtUnixSupport;
+	jfieldID criuSupportLogLevel;
+	jfieldID criuSupportLogFile;
```

Comment:
Perhaps it would clarify things if you group all the fields in support of CRIU in a separate structure (following the pattern in `J9CudaGlobals`?